Source/AwsCommonRuntimeKit/auth/credentials/CRTAWSCredentialsProvider.swift

```diff
@@ -27,7 +27,7 @@ public final class CRTAWSCredentialsProvider {
                                                                 delegate_user_data: credProviderPtr)
 
         guard let credProvider = aws_credentials_provider_new_delegate(allocator.rawValue, &options) else {
-            throw CRTError(errorCode: aws_last_error()) }
+            throw AWSCommonRuntimeError.CRTError(CRTError.init(fromErrorCode: aws_last_error())) }
```

Comment:
.init is not required for all `CRTError.init` calls

Reply:
Thanks, fixed.